kernel/packages/shared/meta/types.ts

```diff
@@ -22,20 +22,21 @@ export type MetaConfiguration = {
 export type WorldConfig = {
   pois: Vector2Component[]
   renderProfile?: RenderProfile
-  messageOfTheDay?: MessageOfTheDayConfig
+  messageOfTheDay?: MessageOfTheDayConfig | null
+  messageOfTheDayInit?: boolean
 }
 
 export type MessageOfTheDayConfig = {
-  imageUrl: string
-  endUnixTimestamp: number
+  background_banner: string
```

Comment:
There's a real reason for using snake_case here? It looks inconsistent with the camelCase of the rest of the payload.

Reply:
The endpoint is set like this and change it is a bit of a mess @marcosnc

Reply:
Ok, If its that difficult to change we can review the naming later then.